src/components/interstitials/Trending.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react'
-import {View} from 'react-native'
-import {ScrollView} from 'react-native-gesture-handler'
```

Comment:
Changed this to RN one but wrapped in GestureDetector. I don't think anything got worse but worth a double check.